test/test_quantized_models.py

```diff
@@ -0,0 +1,26 @@
+import torch
+import torch.jit
+from common_utils import run_tests
+from common_quantization import QuantizationTestCase, ModelMultipleOps
+
+class ModelNumerics(QuantizationTestCase):
+    def test_float_quant_compare(self):
+        torch.manual_seed(42)
+        myModel = ModelMultipleOps().to(torch.float32)
+        myModel.eval()
+        calib_data = torch.rand(1024, 3, 15, 15, dtype=torch.float32)
+        eval_data = torch.rand(1, 3, 15, 15, dtype=torch.float32)
+        out_ref = myModel(eval_data)
+        qModel = torch.quantization.QuantWrapper(myModel)
+        qModel.eval()
+        qModel.qconfig = torch.quantization.default_qconfig
+        torch.quantization.fuse_modules(qModel.module, [['conv1', 'bn1', 'relu1']])
+        torch.quantization.prepare(qModel)
+        qModel(calib_data)
+        torch.quantization.convert(qModel)
+        out_q = qModel(eval_data)
+        SQNRdB = 20 * torch.log10(torch.norm(out_ref) / torch.norm(out_ref - out_q))
+        self.assertGreater(SQNRdB, 30, msg='Quantized model numerics diverge from float, expect SQNR > 30 dB')
```

Comment:
Maybe add a comment to explain the choice of 30 dB here.

Reply:
Will do prior to landing